test-requirements.txt

```diff
@@ -1,2 +1,3 @@
 nose
 mock
+flask
```

Comment:
Please specify a version range for flask. The tests shouldn't fail if flask releases a new major version that causes our flask related tests to fail.